cmd/exo/cmd/request.go

```diff
@@ -48,13 +48,15 @@ func asyncTasks(tasks []task) ([]interface{}, []error) {
 				// decor.DSyncWidth bit enables column width synchronization
 				decor.Percentage(decor.WCSyncSpace),
 			),
-			mpb.AppendDecorators(
-				// replace ETA decorator with "done" message, OnComplete event
-				decor.OnComplete(
-					// ETA decorator with ewma age of 60
-					decor.EwmaETA(decor.ET_STYLE_GO, 60), "done",
-				),
-			),
+
+			//XXX remove for display issue
+			// mpb.AppendDecorators(
+			// 	// replace ETA decorator with "done" message, OnComplete event
+			// 	decor.OnComplete(
+			// 		// ETA decorator with ewma age of 60
+			// 		decor.EwmaETA(decor.ET_STYLE_GO, 60), "done",
+			// 	),
+			// ),
```

Comment:
then remove it.

Reply:
https://github.com/vbauerster/mpb/issues/26